src/Functions/stress_tensor.jl

```diff
@@ -62,12 +62,13 @@ function (f::ElementStressTensor)(u::DisplacementResult; element_dofs=false)
     n_basefuncs = getnbasefunctions(st.cellvalues)
     n_quad = getnquadpoints(st.cellvalues)
     dim = TopOptProblems.getdim(st.problem)
-    return sum(
-        map(1:n_basefuncs, 1:n_quad) do a, q_point
+    return sum(map(1:n_quad) do  q_point
+        dΩ = getdetJdV(st.cellvalues, q_point) 
+        sum(map(1:n_basefuncs) do a   
             _u = cellu[dim * (a - 1) .+ (1:dim)]
             return tensor_kernel(f, q_point, a)(DisplacementResult(_u))
-        end,
-    )
+        end) * dΩ
+    end)
```

Comment:
Shouldn't we divide by the volume of each element at the end? `dΩ` scales up with the volume of the element but the stress returned should be the average stress over the element.

Reply:
Multiplying by what we are calling dΩ here is equivalent to multiplying by the appropriate quadrature weight multiplied by the determinant of the Jacobian which is doing the volume scaling necessary to convert our integral in the bi-unit domain to one in the real space. For instance, in 2D our Jacobian would be dx/dξ = [(h_1)/2 0; 0 (h_2)/2] with x_i  = h_i*(1+ξ_i)/2 (no sum on i). At least for my cuboid meshes h_i = 1 for all dimensions - the determinant of this is just 0.25 which adjusts for the fact that we integrated an area 4 times as large as the real domain because the bi-unit domain always falls between -1 and 1 for each axis instead of 0 to h_i for each axis. Does that make sense? That is at least how it intuitively makes sense to me. I also consulted with the [Ferrite](https://ferrite-fem.github.io/Ferrite.jl/stable/reference/fevalues/#Ferrite.getdetJdV) documentation and it seemed to agree with my method.

Reply:
Definitely let me know if you see my error if I am making one though haha